app/forms/candidate_interface/gcse_qualification_details_form.rb

```diff
@@ -72,13 +72,19 @@ def award_year_is_a_valid_date
     def validate_grade_format
       return if qualification.qualification_type.nil? || qualification.qualification_type == 'other_uk' || qualification.qualification_type == 'non_uk'
 
-      qualification_rexp = invalid_grades[qualification.qualification_type.to_sym]
+      if qualification.qualification_type == 'gcse' && qualification.subject == 'maths'
+        errors.add(:grade, :invalid) unless SINGLE_GCSE_GRADES.include?(sanitize(grade))
+      else
+        qualification_rexp = invalid_grades[qualification.qualification_type.to_sym]
 
-      if qualification_rexp && grade.match(qualification_rexp)
-        errors.add(:grade, :invalid)
+        errors.add(:grade, :invalid) if qualification_rexp && grade.match(qualification_rexp)
       end
     end
 
+    def sanitize(grade)
```

Comment:
Hey @TobyRet, just had a go at this in the review app. I submitted a lowercase grade with a space next to it successfully.
![image](https://user-images.githubusercontent.com/62567622/99562860-86fb2e00-29c0-11eb-8c4e-2bc4b9f6e9ec.png)
![image](https://user-images.githubusercontent.com/62567622/99562882-8cf10f00-29c0-11eb-8045-81abde37fe92.png)
is this intended behaviour?

Should the sanitize method also convert the grade as well as help validate it?

Reply:
Woops I forgot to save the sanitized grade. I'll fix

Reply:
@george-bit - that's all fixed now

https://github.com/DFE-Digital/apply-for-teacher-training/pull/3457/files#diff-821b5429d131177a8a261fc366f7bf33db627f1ac83ae68f4ba9a89ba654d74eR125

Spec added as well